### PR TITLE
Influxdb migrated to PetSet and PersistentVolumes.

### PR DESCRIPTION
**Pull request by jszczepkowski**
``` release-note
Influxdb migrated to PetSet and PersistentVolumes.
```

[Analytics]()

Influxdb migrated to PetSet and PersistentVolumes.

### CODE REVIEW

**Comment by jszczepkowski** [comment]
CC @mwielgus

**Comment by jszczepkowski** [comment]
Resolves https://github.com/kubernetes/heapster/issues/1197.

File: cluster/addons/cluster-monitoring/influxdb/influxdb-claim.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  name: influxdb-claim
+  namespace: kube-system
+  labels:
+    kubernetes.io/cluster-service: "true"
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
```
Comment (piosz) [inline cluster/addons/cluster-monitoring/influxdb/influxdb-claim.yaml]: @mwielgus do you think this value is reasonable?

**Comment by piosz** [comment]
LGTM

File: cluster/addons/cluster-monitoring/influxdb/influxdb-claim.yaml (hunk 1 above)
Comment (mwielgus) [inline cluster/addons/cluster-monitoring/influxdb/influxdb-claim.yaml]: Maybe :)

**Pull request merged**